### ISSUE TITLE
request: Insert the age in the birthday message (If the birth-year is registered)

### ISSUE THREAD

**Issue by CodingFactoryT** [BirthdayyBot/BirthdayyBot#604]
### Is there an existing issue or pull request for this?

- [X] I have searched the existing issues and pull requests

### Feature description

It would be nice if the age was displayed in the birthday message if the user set up his birth-year on birthday registration.

### Desired solution

Include the age in the birthday message

### Alternatives considered

-

### Additional context

-

**Comment by nikolaischunk** [comment]
good idea, unfortunately development on Birthdayy is stagnant for now, since we don't have the capacity to work on it anymore :(
